app/controllers/link_groups_controller.rb

```diff
@@ -2,20 +2,20 @@ class LinkGroupsController < ApplicationController
   before_action :find_link_group, only: [:edit, :update, :destroy]
   before_action :find_channel
   before_action :channel_member?
-  before_action :for_ajax_render_find, only: [:create, :update, :destroy]
-  
+  before_action :ajax_find, only: [:index, :create, :update, :destroy]
+
   def index
-    @link_groups = @channel.link_groups.order(created_at: :asc).includes(:saved_links)
     @link_group = LinkGroup.new
-    @saved_link = SavedLink.new
   end
 
   def new
-    @link_group = LinkGroup.new
   end
 
   def create
-    @link_group          = @channel.link_groups.build(link_group_params)
+    @link_group = @channel.link_groups.build(link_group_params)
+
+    @link_group.position = LinkGroup.all.length * -1
```

Comment:
```suggestion
    @link_group.position = LinkGroup.minimum(:position) - 1
```

https://guides.rubyonrails.org/active_record_querying.html#minimum

Reply:
如果是這樣，改成以下比較保險？
```ruby
@link_group.position = @channel.link_groups.minimum(:position) - 1
```

Reply:
已改